docs/nitro-deploy.md

```
@@ -586,6 +586,103 @@ A restart of this should not be needed but if you need to:
 sudo systemctl restart vsock-apple-proxy.service
 ```
 
+## Vsock APNs and FCM proxies
+Create vsock proxy services so that enclave program can talk to push providers:
+
+First configure the endpoints into their allowlist:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)